wrench/src/rawtest.rs

```diff
@@ -47,6 +48,8 @@ impl<'a> RawtestHarness<'a> {
         self.test_very_large_blob();
         self.test_save_restore();
         self.test_capture();
+        self.test_zero_height_window();
+        self.test_zero_height_window();
```

Comment:
why is it repeated many times?

Reply:
Whoops, sorry, that was from local testing to make sure it wasn't leaving stuff in a bad state. Updated patch so that it only runs once.